command/operator_root.go

```diff
@@ -0,0 +1,31 @@
+package command
+
+import (
+	"strings"
+
+	"github.com/mitchellh/cli"
+)
+
+type OperatorRootCommand struct {
+	Meta
+}
+
+func (*OperatorRootCommand) Help() string {
+	helpText := `
+	Usage: nomad operator root [options]
+
+	This command is accessed by using one of the subcommands below.
+	`
```

Comment:
When running the command locally, the output format seemed slightly off. The whitespace suggestion fixes that and the change to the usage keeps more consistency with other commands and correctly shows subcommand expectation.

```suggestion
Usage: nomad operator root <subcommand> [options] [args]

  This command is accessed by using one of the subcommands below.
`
```